app/helpers/application_helper.rb

```diff
@@ -24,14 +24,17 @@ def currency_dropdown(f: nil, options: [])
 
   def sidebar_link_to(name, path, options = {})
     base_class_names = [ "block", "border", "border-transparent", "rounded-xl", "-ml-2", "p-2", "text-sm", "font-medium", "text-gray-500", "flex", "items-center" ]
+
     hover_class_names = [ "hover:bg-white", "hover:border-alpha-black-50", "hover:text-gray-900", "hover:shadow-xs" ]
     current_page_class_names = [ "bg-white", "border-alpha-black-50", "text-gray-900", "shadow-xs" ]
+    active_class_names = [ "active:bg-gray-200", "active:shadow-xs"]
```

Comment:
```suggestion
    active_class_names = [ "active:bg-gray-200", "active:shadow-xs" ]
```

Reply:
Fixed the issue, but could not get format on save to work correctly. If I may ask, what linter is being used?